src/prog/main.F90

```diff
@@ -1778,6 +1778,18 @@ subroutine parseArguments(env, args, inputFile, paramFile, lgrad, &
          case ('--alpha')
             call set_elprop('alpha')
 
+         case ('--raman')
+            call set_elprop('raman')
```

Comment:
In the current state, only the electrical property is set to `alpha` (via redirect `raman`), but no Hessian calculation is conducted. I.e., no frequencies (and intensities) will be available.
One could add this easily via a call similar to: `call set_runtyp('hess')`.